docs/rollup-api-json.js

```diff
@@ -15,23 +15,20 @@
  * create copies.
  */
 
-const chalk = require("chalk");
 const cpy = require("cpy");
 const fs = require("fs-extra");
 const path = require("path");
 const replace = require("replace-in-file");
 
-const originalPath = path.resolve(process.argv[2]);
-const targetPath = path.resolve(process.argv.length > 3 ? process.argv[3] : originalPath);
-const stagingPath = path.join(targetPath, "_staging");
-const outputPath = path.join(targetPath, "_build");
+const main = async (originalPath, targetPath) => {
```

Comment:
Can we add a doc comment to this explaining what it does and what the parameters mean?